babel.config.js

```diff
@@ -19,7 +19,7 @@ const targets = {
 const config = api => {
     const envConfigs = {
         /**
-         * Watch mode and build:esm partial transpilation mode.
+         * Watch mode and build:esm partial transpiration mode.
```

Comment:
This is not a typo.
Transpiling is a specific term for taking source code written in one language and transforming into another language that has a similar level of abstraction.